test-fixture/src/lib.rs

```diff
@@ -53,7 +53,7 @@ pub fn fixture_init() {
 
 // This needs to be > 2ms to avoid it being rounded to zero.
 // NSS operates in milliseconds and halves any value it is provided.
-pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(10);
+pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(10000);
```

Comment:
This is the real fix.

NSS estimates the RTT.  There is a very small allowance for dropped packets and variations in network delay.  The test you wrote added a bunch of extra delay that looked - at least to NSS - like additional delays.